kqueen_ui/blueprints/manager/views.py

```diff
@@ -13,6 +13,8 @@
 from kqueen_ui.utils.wrappers import superadmin_required
 from slugify import slugify
 
+from ...blueprints.ui.utils import form_page_ranges
```

Comment:
Can we avoid relative imports? it makes impossible test execution not from project directory location